README.md

```diff
@@ -44,25 +46,35 @@ New stamps can be created by calling a standalone `compose()` function.
+* `compose(stampsOrDescriptors...) => stamp` **Creates stamps.** Take any number of stamps or descriptors.
+Return a new stamp that encapsulates combined behavior. If nothing is passed in, it returns an empty stamp.
 
 ### Stamp
 
-* `stamp(baseObject, args...) => objectInstance` **Creates object instances.** Take a base object and any number of arguments. Return the mutated `baseObject` instance back. If no first argument is passed, it uses a new empty object as the base object. Stamps with function or array base objects should not extend the built-in prototypes for `Object` or `Array`.
- * `.compose(stampsOrDescriptors...) => stamp` **Creates stamps.** *a method exposed by all composables (i.e., stamps)* identical to `compose()`, except it prepends `this` to the stamp parameters. Stamp descriptor properties are attached to the `.compose` method., i.e. `stamp.compose.*`
+* `stamp(baseObject, args...) => objectInstance` **Creates object instances.**
+Take a base object and any number of arguments. Return the mutated `baseObject` instance back.
+If no first argument is passed, it uses a new empty object as the base object.
+Stamps with function or array base objects should not extend the built-in prototypes for `Object` or `Array`.
+  * `.compose(stampsOrDescriptors...) => stamp` **Creates stamps.** *a method exposed by all composables (i.e., stamps)*
+    identical to `compose()`, except it prepends `this` to the stamp parameters.
+    Stamp descriptor properties are attached to the `.compose` method., i.e. `stamp.compose.*`
 
 
 ## The Stamp Descriptor
 
-The names and definitions of the fixed properties that form the stamp descriptor. The stamp descriptor properties are made available on each stamp as `stamp.compose.*`
+The names and definitions of the fixed properties that form the stamp descriptor.
+The stamp descriptor properties are made available on each stamp as `stamp.compose.*`
 
 * `methods` - A set of methods that will be added to the object's delegate prototype.
 * `properties` - A set of properties that will be added to new object instances by assignment.
 * `deepProperties` - A set of properties that will be added to new object instances by assignment with deep property merge.
 * `initializers` - A set of functions that will run in sequence and passed the data necessary to initialize a stamp instance.
 * `staticProperties` - A set of static properties that will be copied by assignment to the stamp.
-* `propertyDescriptors` - A set of [object property descriptors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperties) used for fine-grained control over object property behaviors
-* `configuration` - A set of options made available to the stamp and its initializers during object instance creation. Configuration properties get deep merged.
+* `propertyDescriptors` - A set of [object property
```

Comment:
It's called `propertyDescriptors` because they are called "property descriptors" in [the ECMAScript specification](http://www.ecma-international.org/ecma-262/5.1/#sec-8.10) & [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty).